libs/ui-components/src/components/form/NumberField.tsx

```
@@ -0,0 +1,78 @@
+      doChange(field.value ? field.value - 1 : min);
+    };
+
+    const handleChange: PFNumberInputProps['onChange'] = (event: React.FormEvent<HTMLInputElement>) => {
+      const valStr = (event.target as HTMLInputElement).value;
+      // Skip updating the value onBlur. Otherwise, when the field is cleared PF sets a numeric value onBlur.
+      if (event.type === 'change') {
+        if (!isRequired && valStr === '') {
+          setValue('');
+        } else {
+          const targetValue = valStr as unknown as number;
+          doChange(isNaN(targetValue) ? min : Number(targetValue));
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)